src/graphics/EInkDisplay2.cpp

```diff
@@ -221,6 +243,43 @@ bool EInkDisplay::connect()
             (void)adafruitDisplay;
         }
     }
+
+#elif defined(HELTEC_WIRELESS_PAPER_V1_0)
+    {
+        // Is this a normal boot, or a wake from deep sleep?
+        esp_sleep_wakeup_cause_t wakeReason = esp_sleep_get_wakeup_cause();
+
+        // If waking from sleep, need to reverse rtc_gpio_isolate(), called in cpuDeepSleep()
```

Comment:
This may also be relevant for other E-Ink displays?
https://github.com/meshtastic/firmware/blob/4c55d5d9e46eeb7ca5c1ab22c9e5b75a9cc9a9a7/src/graphics/EInkDisplay2.h#L13

Reply:
yes, but the nrf52 target dont sleep at all